pkg/ctl/get/addon.go

```diff
@@ -47,19 +47,16 @@ func getAddonCmd(cmd *cmdutils.Cmd) {
 }
 
 func getAddon(cmd *cmdutils.Cmd, params *getCmdParams) error {
+	if params.output != printers.TableType {
+		//log warnings and errors to stdout
+		logger.Writer = os.Stderr
+	}
```

Comment:
this pattern seems to be used a lot in the `get` commands for when we want `json/yaml` output, basically ensures `get addons | jq .` still works, but the user still sees our helpful log messages on `stderr`